### PR TITLE
Ci

### PR DESCRIPTION
**Pull request by mpaauw**
Should run builds here: https://travis-ci.org/mpaauw/aoc-collab

cc @Cryowatt

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+notifications:
+  email: false
+
+language: csharp
+mono: none
+dotnet: 2.1
```
Comment (shawnkoon) [inline .travis.yml]: How about try using ?

```suggestion
dotnet: 2.2
sudo: required
dist: xenial
```

I had some problem with 2.2 but `dist: xenial` fixed it https://travis-ci.community/t/dotnet-core-2-2/1216/10

Comment (mpaauw) [reply .travis.yml]: 🤞

Comment (mpaauw) [reply .travis.yml]: @shawnkoon ++

**Comment by Cryowatt** [comment]
> Looks good with me, should we add *.sln to the .gitignore?

Naw, *.sln is used buy `dotnet build` to identify projects to build. Totes-required.

**Comment by mpaauw** [comment]
🤔 @shawnkoon with travis, do you know if we can support multiple languages in a single repo? Would we just need multiple travis yml's in that case? I'm considering writing my code in Java.

**Comment by shawnkoon** [comment]
@mpaauw Yeap 👍 very similar to setting up `docker-compose.yml` file, we prob have to update repo structure into something like this to keep things organized tho.

```bash
$ tree

aoc-collab
  ㄴ dotnet
      ㄴ ...
  ㄴ python
      ㄴ ...
  ㄴ java
      ㄴ ...
  ㄴ .gitignore
  ㄴ .travis.yml
  ㄴ ...
```

You can find out more info about this here : 
- https://docs.travis-ci.com/user/customizing-the-build/#build-matrix
- https://stackoverflow.com/questions/27644586/how-to-set-up-travis-ci-with-multiple-languages?answertab=votes#tab-top

**Pull request merged**